dist/infiniband-radar-daemon.spec

```diff
@@ -0,0 +1,67 @@
+# run "spectool -g -R infiniband-radar-daemon.spec" to automatically download 
+# source files. spectool is part of the rpmdevtools package
+
+%define use_systemd (0%{?fedora} && 0%{?fedora} >= 18) || (0%{?rhel} && 0%{?rhel} >= 7) || (0%{?suse_version} && 0%{?suse_version} >=1210)
+
+Name:		infiniband-radar-daemon
+Version:	2.07
```

Comment:
Is there a way to somehow automate the versioning?
If this pull request is being merged it will probably be 2.09 already.

Reply:
That may be possible, but where should the version number be taken from? 
RPMs will not be necessarily be built form a cloned Git repository, they could be build from a downloaded release archive, for instance, in which case there will not be any Git-related information.